src/distilabel/steps/generators/huggingface.py

```diff
@@ -387,6 +405,11 @@ def get_filetype(data_path: UPath) -> str:
         if data_path.is_file():
             filetype = get_filetype(data_path)
             data_files = str(data_path)
+
+        elif len(glob.glob(str(data_path))) >= 1:
```

Comment:
I think `glob` package won't work with `s3://...` or other remote file systems. I suggests the following:

```python
parent = data_path.parent
len(parent.glob(data_path.name))
```

Reply:
my bad!